schema-guide.md

```diff
@@ -193,104 +205,174 @@ commit: "Revision: 8612"
 
 Usage example:
 
 ```yaml
+date-released: 2020-01-31
 ```
 
 ### `doi`
 
-- type: ...
-- required: false
+- type: [`definitions.doi`](#definitionsdoi)
+- required: `false`
 
-...
+The DOI of the software or data set. This notation is most useful when there is just one DOI you want to include. In
+that case, `doi` can be used as shorthand for something like:
 
-Usage example:
+```yaml
+identifiers:
+  - type: doi
+    value: 10.5281/zenodo.1003149
+    description: The concept DOI of the work.
+```
+or
 
 ```yaml
+identifiers:
+  - type: doi
+    value: 10.5281/zenodo.4813122
+    description: The versioned DOI of the work.
 ```
 
-### `identifiers`
+Usage examples:
 
-- type: ...
-- required: false
+```yaml
+doi: 10.5281/zenodo.1003149
+```
+```yaml
+doi: 10.5281/zenodo.4813122
+```
 
-...
+### `identifiers`
 
-Usage example:
+- type: array of [`definitions.identifier`](#definitionsidentifier) objects.
+- required: `false`
 
-```yaml
-```
+The identifiers of the software or dataset.
+
+Usage example: see [`definitions.identifier`](#definitionsidentifier).
 
 ### `keywords`
 
-- type: ...
-- required: false
+- type: array of `string`
+- required: `false`
 
-...
+Keywords that describe the work.
 
 Usage example:
 
 ```yaml
+keywords:
+ - thefirstkeyword
+ - thesecondkeyword
+ - and a third
 ```
 
 ### `license`
 
-- type: ...
-- required: false
+- type: (array of) [`definitions.license-enum`](#definitionslicense-enum).
+- required: `false`
 
-...
+The SPDX license identifier(s) for the license(s) under which the work is made available. When there are multiple
+licenses, it is assumed their relationship is OR, not AND.
 
-Usage example:
+Usage examples:
 
 ```yaml
+license: Apache-2.0
+```
+```yaml
+license:
+ - Apache-2.0
+ - MIT
+```
+```yaml
+license:
+ - GPL-3.0
+ - GPL-3.0-or-later
 ```
 
 ### `license-url`
 
-- type: ...
-- required: false
+- type: [`definitions.url`](#definitionsurl)
+- required: `false`
 
-...
+The URL of the license text under which the software or dataset is licensed (only for non-standard licenses not included in the SPDX License List).
 
 Usage example:
 
 ```yaml
+license-url: "https://obscure-licenses.com?id=1234"
 ```
 
 ### `message`
 
-- type: ...
-- required: true
+- type: `string`
+- required: `true`
+- default: `If you use this software, please cite it using the metadata from this file.`
 
-...
+A message to the human reader of the CITATION.cff file to let them know what to do with the citation metadata.
 
-Usage example:
+Usage examples:
 
 ```yaml
+message: If you use this software, please cite it using the metadata from this file.
+```
+```yaml
+message: Please cite this software using these metadata.
+```
+```yaml
+message: Please cite this software using the metadata from 'preferred-citation'.
+```
+```yaml
+message: If you use this data set, please cite it using the metadata from this file.
```

Comment:
```suggestion
message: If you use this dataset, please cite it using the metadata from this file.
```

[CED](https://dictionary.cambridge.org/dictionary/english/dataset) says one word, although [APA Style Blog](https://blog.apastyle.org/apastyle/2012/07/data-is-or-data-are.html) says differently (but who are they in the world of lexicographers!), so I asked the world corpus Google Search which gives 64M hits for `"data set"`and 125M for `"dataset"`. Clear result if you ask me.

Reply:
good enough for me :)